docs/tutorials/LCD.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Using the LCD
+subject: LCD
+---
+
+* Table of Contents
+{:toc}
+
+NOTE: You need to be a member of the `video` group to use the LCD
+
+## Basics
+The EV3 has a 178 x 128 pixels monochrome LCD. The ev3dev video driver
+provides [standard Linux framebuffer interface](https://www.kernel.org/doc/Documentation/fb/api.txt)
+and it's possible to write pixel data into `/dev/fb0` directly using `write`
+or `mmap`.
+
+## The pixel buffer format
+Each byte in the buffer represents 8 pixels. Leftmost pixel is in the least
+significant bit. Value 0 is white and value 1 is black (`fb_fix_screeninfo#visual`
+is set to `FB_VISUAL_MONO01`).
+
+Each row need to be 4 bytes aligned. So it's (178+31)/324 = 24 bytes per row
```

Comment:
This was all cut-n-paste but yeah the math here is hosed.  How about:

Each row needs to be aligned on a 4-byte boundry.  The closest 4-byte boundry to 178 is 192. 192/8 = 24 bytes per row